pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -1045,13 +999,6 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, ReentrancyG
         return _totalTokensCache;
     }
 
-    function _getTokenData(IERC20 token) private view returns (bytes32 tokenData) {
```

Comment:
`getTokenData` was just there for the error checking. If you call getScalingFactor with an invalid token,  it would return 1 (kind of ok). But if you call getTokenWeight with a zero bytes32, it would return zero. It would probably revert downstream anyway, but it might be less obvious why.

Reply:
This is fine, we never call into this function with a user provided token so this is fine.